server/pbenchinacan/load_keycloak.sh

```diff
@@ -19,6 +19,7 @@
 
 KEYCLOAK_HOST_PORT=${KEYCLOAK_HOST_PORT:-"https://localhost:8090"}
 KEYCLOAK_REDIRECT_URI=${KEYCLOAK_REDIRECT_URI:-"https://localhost:8443/*"}
+KEYCLOAK_DEV_ORIGIN=${KEYCLOAK_DEV_ORIGIN:-"http://localhost:3000"}
```

Comment:
For `KEYCLOAK_DEV_ORIGIN` we're adding the wildcard at the reference, while for `KEYCLOAK_REDIRECT_URI` we're adding it in the definition.  Can we treat them both the same way?

Reply:
One of the things I was trying to clean up when the rebase on your change broke my universe. 😆 

Originally I thought I'd have to set the "web origin" separately, but Keycloak derives that from the redirects. So, yeah, I'd already put the `/*` back up top but I'm having trouble *testing* it.